src/applications/personalization/dashboard-2/components/Dashboard.jsx

```diff
@@ -119,11 +119,13 @@ const Dashboard = ({
                 My VA
               </h1>
 
-              <div className="vads-l-row">
-                <div className="vads-l-col--12 medium-screen:vads-l-col--8">
-                  {showHealthCareError && <HealthCareLoadError />}
+              {showHealthCareError ? (
```

Comment:
this is to keep an invisible div from always rendering. that little sliver prevents the margins between the H1 and H2 from collapsing as they should:
<img width="1131" alt="Screen Shot 2021-04-05 at 3 47 12 PM" src="https://user-images.githubusercontent.com/20728956/113638625-2184a280-962c-11eb-844e-7bacb83048d3.png">

Reply:
Thanks for catching that!